.github/workflows/gosec.yml

```diff
@@ -0,0 +1,24 @@
+name: gosec
+
+on:
+  pull_request: {}
+  push: {}
+  schedule:
+    - cron: '0 5 * * *' # Daily at 05:00 UTC
```

Comment:
Hm, why do we need to run this on a schedule in addition to when the code changes?

Reply:
@gicmo told me to

Reply:
Coverity has a limit of three scans per day, we can't run it on every code change sadly :/

Reply:
This is unrelated to coverty. But @gicmo just explained out of band that having this on a schedule can makes sense, because of the usual PR race. Let's leave it. Thanks!